docs/SRS-Volere/SRS.tex

```diff
@@ -6,6 +6,8 @@
 % Risks are removed because they are covered by the Hazard Analysis
 \documentclass[12pt]{article}
 
+\usepackage{graphicx} 
+\graphicspath{ {./images/} }
```

Comment:
I'm realizing now that Allan and I have multiple PRs that add images, and our path is ./res. I think either we have to change to your path or you have to change to ours to avoid problems with images during merge. Simply because Allan and I have like 3 PRs collectively that use ./res, it might be easier for you to change the line and file path in your PR.

Reply:
yea probably change this i think